app/templates/base-cy.html

```diff
@@ -110,29 +110,14 @@
 {%- endblock -%}
 
 {%- block preHeader -%}
+    {% set cookiesPageURL = domain_url_en + '/cy/cookies/' %}
 
     {%- from 'components/cookies-banner/_macro.njk' import onsCookiesBanner -%}
-    {{
-        onsCookiesBanner({
-            'lang': 'cy'
-        })
-    }}
+    {{ onsCookiesBanner({
+        'lang': 'cy',
+        'settingsLinkURL': cookiesPageURL,
+        'statementText': '<p>Ffeiliau bach a gaiff eu storio ar eich dyfais pan fyddwch yn mynd ar wefan yw cwcis. Rydym ni’n defnyddio rhai cwcis hanfodol i wneud i’r wefan hon weithio.</p><p>Hoffem osod <a href="' + cookiesPageURL + '">cwcis ychwanegol</a> er mwyn cofio eich gosodiadau a deall sut rydych chi’n defnyddio’r wefan. Mae hyn yn ein helpu ni i wella ein gwasanaethau.</p>',
+        'preferencesText': 'Gallwch chi <a href="' + cookiesPageURL + '">newid eich dewisiadau o ran cwcis</a> ar unrhyw adeg.'
```

Comment:
These look like they still need to marked `PLACEHOLDER WELSH`?

Reply:
Had a chat, this is the translation copied from the design system so shouldn't be marked for us to translate. Deserves a comment to this effect though